redis_go1.7.go

```diff
@@ -0,0 +1,35 @@
+// +build go1.7
```

Comment:
Add go1.8 too and rename to `redis_context.go`? go1.8 should be released really soon.